lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -491,6 +495,13 @@ export class ClientAuthError extends AuthError {
         return new ClientAuthError(ClientAuthErrorMessage.noAuthorizationCodeFromServer.code, ClientAuthErrorMessage.noAuthorizationCodeFromServer.desc);
     }
 
+    /**
+     * Throws error when no azure region is detected and fallback is provided
+     */
+    static createNoAzureRegionDetectedError(): ClientAuthError {
```

Comment:
Where is this used?

Reply:
This was previously used before the API changed, removing it as it is no longer used.